app.js

```diff
@@ -270,4 +270,4 @@ Vue.createApp({
     await new Promise(resolve => setTimeout(resolve, 400))
     document.querySelector(".loading").remove()
   }
-}).mount("main")
\ No newline at end of file
+}).mount("main")
```

Comment:
Oops, looks like my local git config caused git to add this newline automatically. I can try to change that if you care about it.